packs/octopusdeploy/actions/create_release.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: create_release
+runner_type: run-python
```

Comment:
Yeah, good catch.

We still support old names for backward compatibility reasons, but new code should use `python-script` as you said :)